pkg/kubectl/resource_printer.go

```diff
@@ -380,7 +380,11 @@ func interpretContainerStatus(status *api.ContainerStatus) (string, string, stri
 		if message != "" {
 			message = "last termination: " + message
 		}
-		return "Running", translateTimestamp(state.Running.StartedAt), message, nil
+		stateMsg := "Running"
+		if !status.Ready {
+			stateMsg = stateMsg + " *not ready*"
```

Comment:
would it be useful to bubble this up to the pod, in addition to having it per container?